### ISSUE TITLE
Clicking quit in tray while dialog open should quit app without having to quit dialog

### ISSUE THREAD

**Issue by mkg20001** [Siderus/Orion#156]
**Describe the bug**
Clicking quit in tray while orion asks whether one wants to connect to a node with a different version does not quit the app

**To Reproduce**
0. Run ipfs node
1. Open the app (fresh install)
2. Accept tos
3. Existing node detected. Connect? => yes
4. Node runs different version. Connect anyway? => just click quit.
5. App does not quit

**Expected behavior**
Should instead just quit.

**Screenshots**
![before click](https://user-images.githubusercontent.com/7735145/42759766-73db4e78-8908-11e8-9c8d-e0ab239effef.png)
![after click quit](https://user-images.githubusercontent.com/7735145/42759776-78869b4e-8908-11e8-9423-55c448e2b196.png)

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu 16.04 amd64
 - Browser [e.g. chrome, safari] N/A
 - Version [e.g. 22] orion v0.7

**Additional context**
Logs:
```
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
Checking for update
Error: Error: ENOENT: no such file or directory, open '/opt/Orion/resources/app-update.yml'
    at /opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:391:27
    at Generator.next (<anonymous>)
From previous event:
    at AppImageUpdater.loadUpdateConfig (/opt/Orion/resources/app/node_modules/electron-updater/out/AppUpdater.js:555:7)
    at Lazy.AppUpdater.configOnDisk (/opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:134:43)
    at Lazy.get value [as value] (/opt/Orion/resources/app/node_modules/lazy-val/src/main.ts:18:23)
    at /opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:298:46
From previous event:
    at AppImageUpdater.getUpdateInfo (/opt/Orion/resources/app/node_modules/electron-updater/out/AppUpdater.js:460:7)
    at /opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:308:35
    at Generator.next (<anonymous>)
From previous event:
    at AppImageUpdater.doCheckForUpdates (/opt/Orion/resources/app/node_modules/electron-updater/out/AppUpdater.js:502:7)
    at /opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:279:25
    at Generator.next (<anonymous>)
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
From previous event:
    at AppImageUpdater._checkForUpdates (/opt/Orion/resources/app/node_modules/electron-updater/out/AppUpdater.js:433:7)
    at AppImageUpdater.checkForUpdates (/opt/Orion/resources/app/node_modules/electron-updater/src/AppUpdater.ts:217:35)
    at startOrion (/opt/Orion/resources/app/app/index.js:148:32)
    at <anonymous>
Loading window ready to show
Another service on localhost:5001 has been deteced
Gtk-Message: GtkDialog mapped without a transient parent. This is discouraged.
```

**Comment by mkg20001** [comment]
Additionally if quit was already clicked and the user clicks yes afterwards it quits after completing initialization

**Comment by koalalorenzo** [comment]
I see.
Would it fix the problem if we show the tray icon only AFTER the initialisation is done?

**Comment by mkg20001** [comment]
This would likely fix the problem, although the solution seems a bit like "there is nothing to see here" :D

**Comment by koalalorenzo** [comment]
I have reported the issue and we will work on it :)
I have no estimate as we are doing holidays now :O

**Comment by koalalorenzo** [comment]
Fixed in #170
